cmd/configschema/go.mod

```diff
@@ -620,48 +620,48 @@ require (
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/extension v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/extension/auth v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/extension/ballastextension v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/extension/zpagesextension v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/featuregate v1.0.2-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/pdata v1.0.2-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/processor v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/processor/batchprocessor v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/semconv v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/collector/service v0.92.1-0.20240118172122-8131d31601b8 // indirect
-	go.opentelemetry.io/contrib/config v0.1.1 // indirect
+	go.opentelemetry.io/collector v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configauth v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configcompression v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configgrpc v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/confighttp v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/confignet v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configopaque v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configretry v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/configtelemetry v0.93.0 // indirect
+	go.opentelemetry.io/collector/config/internal v0.93.0 // indirect
+	go.opentelemetry.io/collector/confmap v0.93.0 // indirect
+	go.opentelemetry.io/collector/connector v0.93.0 // indirect
+	go.opentelemetry.io/collector/connector/forwardconnector v0.93.0 // indirect
+	go.opentelemetry.io/collector/consumer v0.93.0 // indirect
+	go.opentelemetry.io/collector/exporter/debugexporter v0.93.0 // indirect
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.93.0 // indirect
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.93.0 // indirect
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.93.0 // indirect
+	go.opentelemetry.io/collector/extension v0.93.0 // indirect
+	go.opentelemetry.io/collector/extension/auth v0.93.0 // indirect
+	go.opentelemetry.io/collector/extension/ballastextension v0.93.0 // indirect
+	go.opentelemetry.io/collector/extension/zpagesextension v0.93.0 // indirect
+	go.opentelemetry.io/collector/featuregate v1.0.1 // indirect
+	go.opentelemetry.io/collector/pdata v1.0.1 // indirect
```

Comment:
I'm just curious, not blocking this change, why pdata change from v1.0.2 to an older version v1.0.1

Reply:
There have been no changes on pdata or featuregate in the last release cycle, so there is no real difference between v1.0.1 and the pseudoversion that was before. The v1.0.2-<commit sha> is not really v1.0.2, it's just [the way Go handles pseudoversions](https://go.dev/ref/mod):

> Each pseudo-version may be in one of three forms, depending on the base version. These forms ensure that a pseudo-version compares higher than its base version, but lower than the next tagged version.
> vX.Y.(Z+1)-0.yyyymmddhhmmss-abcdefabcdef is used when the base version is a release version like vX.Y.Z. For example, if the base version is v1.2.3, a pseudo-version might be v1.2.4-0.20191109021931-daa7c04131f5.